test/dataframerow.jl

```diff
@@ -7,50 +7,109 @@ module TestDataFrameRow
                    c=["A", "B", "C", "A", "B", missing],
                    d=CategoricalArray([:A, missing, :C, :A, missing, :C]))
     df2 = DataFrame(a = [1, 2, 3])
+    sdf = view(df, [5, 3], [3, 1, 2])
+
+    @test names(DataFrameRow(df, 1, :)) == [:a, :b, :c, :d]
+    @test names(DataFrameRow(df, 3, [3, 2])) == [:c, :b]
+    @test copy(DataFrameRow(df, 3, [3, 2])) == (c = "C", b = 1.2)
+    @test copy(DataFrameRow(sdf, 2, [3, 2])) == (b = 1.2, a = 3)
+    @test copy(DataFrameRow(sdf, 2, :)) == (c = "C", a = 3, b = 1.2)
+    @test DataFrameRow(df, 3, [3, 2]) == df[3, [3, 2]] == view(df, 3, [3, 2])
```

Comment:
Use `===`?

Reply:
It will fail as `[3,2]` is mutable. I will add a test for `===` that goes through.